tools/genjava.js

```diff
@@ -36,6 +36,7 @@ var [argv, X, flags] = require('./processArgs.js')(
   },
   {
     buildlib:      false, // generate Maven pom.xml
+    checklib:      false, // generate Maven pom.xml for checking dependencies vulnerabilities
```

Comment:
Going forward, I don't think we need to add new features to genjava and genjs as I hope to remove them soon.